instagram-app/src/components/CommentSection/CommentSection.js

```diff
@@ -1,15 +1,50 @@
 import React from "react";
 import Comment from "./Comment";
 
-const CommentSection = props => {
+class CommentSection extends React.Component {
   //   console.log(props.comments);
-  return (
-    <div className="comments-section">
-      <p>
-        <Comment comments={props.comments} />
-      </p>
-    </div>
-  );
-};
+  state = {
+    comments: this.props.comments,
+    newComment: ""
```

Comment:
```suggestion
    newComment: {}
```
better to initialize this as an object if that's what it is going to be